Makefile

```diff
@@ -82,6 +82,7 @@ export GIT_PAGER :=
 TAR_XFORM_FLAG = $(shell $(TAR) --version | grep -q GNU && echo "--xform=s" || echo "-s")
 
 GIT_DIR := $(shell git rev-parse --git-dir 2> /dev/null)
+LDFLAGS += -s -w
```

Comment:
Maybe this should go in the `TYPE=release` section, so we still get debug symbols in regular builds? Or do stripped binaries still contain enough debug info for the stuff we care about? (for me, that's mainly `pprof`)